ci/pipeline.yml

```diff
@@ -12,115 +11,116 @@ groups:
       - shipit
       - major
       - minor
+      - autobump-dependencies
 
 jobs:
   - name: unit-tests
     public: true
     serial: true
     plan:
-    - do:
```

Comment:
shouldn't this change only add
- autobump-job
- daily trigger
?
seems like there is a whole lot of useless whitespace formatting going on here...